package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protocol-registry",
-  "version": "1.3.6",
+  "version": "1.3.7",
```

Comment:
Please don't change the version number It will be changes while publishing the module.